### PR TITLE
(#1088) Improve test coverage

### PR DESCRIPTION
**Pull request by baudoliver7**
For #1088

- Improve test coverage of packages xe, fork and tk

### CODE REVIEW

**Review by andreoss — CHANGES_REQUESTED** [review]

File: src/test/java/org/takes/rs/xe/XeWhenTest.java [hunk 1]
```diff
@@ -124,6 +125,23 @@ public XeSource value() throws IOException {
                 "/negative/memory"
             )
         );
+        MatcherAssert.assertThat(
+            IOUtils.toString(
```
[the remark this answers is not in the document]
Comment (baudoliver7) [reply src/test/java/org/takes/rs/xe/XeWhenTest.java:131]: @andreoss Done.

**Comment by baudoliver7** [comment]
@andreoss Please, see my new changes.

**Comment by andreoss** [comment]
@baudoliver7 Sorry, I meant using cactoos instead of `IOUtils`.

**Review by andreoss — APPROVED** [review]

**Comment by baudoliver7** [comment]
@victornoel ping

**Comment by victornoel** [comment]
@baudoliver7 I'm not involved in takes :)

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/takes/issues/1094#issuecomment-803908494))

@victornoel This is what I know about this job in C3FFK3YAY, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1094](https://github.com/yegor256/takes/issues/1094) is [in scope](http://www.zerocracy.com/policy.html#14) for 10days
  * The role is `REV`
  * The job is assigned to @andreoss[/z](https://www.0crat.com/u/andreoss) for [10days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @baudoliver7[/z](https://www.0crat.com/u/baudoliver7): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3FFK3YAY?q=%7Bjob%3A%27gh%3Ayegor256%2Ftakes%231094%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by baudoliver7** [comment]
@victornoel Excuse me. it was a mistake. This ping was for @paulodamaso.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by baudoliver7** [comment]
@yegor256 could you check/merge this PR please ? It has been approved by @andreoss.

**Comment by baudoliver7** [comment]
@yegor256 Please, could you merge now ? Previous mergers failed because of rultor troubles. But it has been solved.

**Comment by baudoliver7** [comment]
@yegor256 please, have a look on my suggestion.

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@baudoliver7 check your code, it seems that the defect is there

**Comment by baudoliver7** [comment]
@yegor256 fixed now. Could you try again ?

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@baudoliver7 thanks!

**Comment by 0crat** [comment]
@andreoss[/z](https://www.0crat.com/u/andreoss) resigned from [#1094](https://github.com/yegor256/takes/issues/1094), since the job is not in scope anymore

**Pull request merged**